yazi-config/preset/keymap.toml

```diff
@@ -232,10 +232,10 @@ keymap = [
 	{ on = "<End>",  run = "move 999",  desc = "Move to the EOL" },
 
 	# Delete
-	{ on = "<Backspace>", run = "backspace",	       desc = "Delete the character before the cursor" },
-	{ on = "<Delete>",    run = "backspace --under", desc = "Delete the character under the cursor" },
-	{ on = "<C-h>",       run = "backspace",         desc = "Delete the character before the cursor" },
-	{ on = "<C-d>",       run = "backspace --under", desc = "Delete the character under the cursor" },
+	{ on = "<Backspace>", run = "backspace --close",         desc = "Delete the character before the cursor" },
```

Comment:
This is a breaking change that will alter the behavior for users with just `backspace --under` in their existing config. 

Also could you explain the necessity of adding it? "close when there's no more content" is meant to better mimic the behavior of the Vim command line prompt.

Reply:
When I want to delete all content of an `input`, I usually hold `backspace` for a while. And then I will to input something, but the `input` has already been closed, all my inputs will act as keymap.

Reply:
What about change this option to `--no-close` to make it not break?